### ISSUE TITLE
[Components and pattern standards] Your content or feature doesn't use standard global navigation components or is missing a standard navigation component.  (04.02.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#59487]
#### VFS team name
MHV

#### VFS product name
My HealtheVet

#### VFS feature name
Secure Messaging

#### Point of Contact/Reviewers
Sara Sterkenburg - @sterkenburgsara - IA

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Your content or feature doesn't use standard global navigation components or is missing a standard navigation component. 

### Issue Details
There are no back breadcrumbs when a user enters into any individual messages on any page of the tool. There are also no back breadcrumbs on the interstitial pages. Back breadcrumbs are an important navigational component that allows a user to flow across and out of pages of this experience. At present, the only escape hatch out of a particular message in "drafts" for instance (or any other page) is to use the sidenav.

### VA.gov Experience Standard
[Category Number 04, Issue Number 02](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
Add back breadcrumbs to child pages throughout this product.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by bryan-riley-va** [comment]
JIRA ticket created: https://jira.devops.va.gov/browse/MHV-45984

**Comment by mnorthuis** [comment]
@bryan-riley-va please tag whoever is appropriate for this work. 

Per conversations with @coultonbunney-usds, here are some details around how the breadcrumbs on the new message and message thread screens should look and function. This includes the interstitials. 

Start new message
- Consider this a 2 step flow - interstitial page and new message page
- Breadcrumb on both pages should be"  "< Back to inbox"
- When you discard from the new message page, you should be returned to the inbox, never the interstitial

Message thread/reply
- Consider this up to a 3 step flow - view message thread, interstitial page, reply to message
- Breadcrumb on all steps should be "< Back to [previous view]" where it pulls in where the user came from when they accessed the message thread/reply/draft (i.e. "Back to drafts")
- When you discard from the reply page, you should be returned to the view message thread, never the interstitial

Dropping an image of the flow below, [linking to the Mural where this is mapped out for reference](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1671138122441/c6eae887e8c0467e616d61002f14085580dfb59c?wid=21-1685654779687) - please reach out with any questions

![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/20994159/843611ee-44c8-457c-9989-28a59426fd16)

**Comment by vsaleem** [comment]
The changes for this concern are merged into staging and have been validated by @mnorthuis and @AngelaFowler82

**Comment by vsaleem** [comment]
https://dsva.slack.com/archives/C03ECSBGSKX/p1686163246090049

**Comment by shiragoodman** [comment]
FYSA @sterkenburgsara
